scripts/clone_template.py

```diff
@@ -178,11 +178,15 @@ def main(**kwargs):
         provider.deploy_template(**deploy_args)
     except Exception as e:
         logger.exception(e)
-        logger.error('Clone failed')
-        if kwargs.get('cleanup', None):
+        logger.error('provider.deploy_template failed')
```

Comment:
Here and the new block below are the core of the PR, everything else is CQ cleanup.